apps/meteor/app/authentication/server/startup/index.js

```diff
@@ -157,8 +157,34 @@ const getLinkedInName = ({ firstName, lastName }) => {
 	return `${firstName} ${lastName}`;
 };
 
+const validateEmailDomain = (user) => {
+	if (user.type === 'visitor') {
+		return true;
+	}
+
+	let domainWhiteList = settings.get('Accounts_AllowedDomainsList');
+	if (_.isEmpty(domainWhiteList?.trim())) {
```

Comment:
DDP Streamer has an isEmpty function, **ee/apps/ddp-streamer/src/lib/utils.ts**

Not sure if moving that to the `utils` package and using it would be better here.

Reply:
This piece is not new code, I just moved it from a callback to a function. I intend to remove it completely in a subsequent PR but for this release I'm not changing the implementation.